src/theme/colors/colors.ts

```diff
@@ -156,6 +156,16 @@ export const red = {
   10: '#660624'
 };
 
+export const redOrange = {
```

Comment:
@surajgjadhav can you tell from where you added these hex codes.

Reply:
from the link mentioned din original issue #603
https://maketintsandshades.com/#B32700